### ISSUE TITLE
jfxRun gives weld exeption

### ISSUE THREAD

**Issue by lgLindstrom** [FibreFoX/javafx-gradle-plugin#117]
Hi
I am writing a JavaFx application.
Gradle 4.4.1
JavaFx 8.8.2
Java 9

From Intellij (2018.1) terminal:
    $gradle run
works nicely.

From terminal:
   $gradle jfxRun

give a exeption that have origin in Weld.

Transcript below.

```
C:\HostedProjects\PurchaseListGenerator>gradle jfxRun

> Task :jfxRun FAILED
Running JavaFX Application
Running command: java -jar build/libs/purchaseList.jar
Exception in Application init method
java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:564)
        at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:473)
        at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:372)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:564)
        at java.base/sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:945)
Caused by: java.lang.RuntimeException: Exception in Application init method
        at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:968)
        at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:198)
        at java.base/java.lang.Thread.run(Thread.java:844)
Caused by: java.lang.NoClassDefFoundError: org/jboss/weld/environment/se/Weld
        at com.consius.purchaseList.MainApplication.init(MainApplication.java:42)
        at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:897)
        ... 2 more
Caused by: java.lang.ClassNotFoundException: org.jboss.weld.environment.se.Weld
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:185)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
        ... 4 more
Exception running application com.consius.purchaseList.MainApplication


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jfxRun'.
> There was an exception while executing JavaFX Application. Please check build-log.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s
5 actionable tasks: 2 executed, 3 up-to-date
 
```

This is my jfx configuration:

```
// configure javafx-gradle-plugin
jfx {
    verbose = true
    mainClass = "com.consius.purchaseList.MainApplication"
    jfxAppOutputDir = "build/jfx/app"
    jfxMainAppJarName = "build/libs/purchaseList.jar"
    deployDir = "src/main/deploy"

    // gradle jfxJar
    css2bin = false
    preLoader = null
    updateExistingJar = false
    allPermissions = false
    manifestAttributes = null // Map<String, String>
    addPackagerJar = true

    // gradle jfxNative
    identifier = null // setting this for windows-bundlers makes it possible to generate upgradeable installers (using same GUID)
    vendor = "Consius AB"
    nativeOutputDir = "build/jfx/native"
    bundler = "ALL" // set this to some specific, if your don't want all bundlers running, examples "windows.app", "jnlp", ...
    jvmProperties = null // Map<String, String>
    jvmArgs = null // List<String>
    userJvmArgs = null // Map<String, String>
    launcherArguments = null // List<String>
    nativeReleaseVersion = "0.4"
    needShortcut = true
    needMenu = true
    //bundleArguments = [
    //        // dont bundle JRE (not recommended, but increases build-size/-speed)
    //        runtime: null
    //]
    appName = "Purchase List Generator" // this is used for files below "src/main/deploy", e.g. "src/main/deploy/windows/settings.ico"
    additionalAppResources = null // path to some additional resources when creating application-bundle
    secondaryLaunchers = []
    fileAssociations = null // List<Map<String, Object>>
    noBlobSigning = false // when using bundler "jnlp", you can choose to NOT use blob signing

    skipNativeLauncherWorkaround124 = false
    skipNativeLauncherWorkaround167 = false
    skipJNLPRessourcePathWorkaround182 = false
    skipSigningJarFilesJNLP185 = false
    skipSizeRecalculationForJNLP185 = false

    // gradle jfxGenerateKeyStore
    keyStore = "src/main/deploy/keystore.jks"
    keyStoreAlias = "myalias"
    keyStorePassword = "password"
    keyPassword = null // will default to keyStorePassword
    keyStoreType = "jks"
    overwriteKeyStore = false

    certDomain = null // required
    certOrgUnit = null // defaults to "none"
    certOrg = null // required
    certState = null // required
    certCountry = null // required
}
```

**Comment by FibreFoX** [comment]
Version 8.8.2 is not yet compatible to JDK 9+

**Comment by FibreFoX** [comment]
Does it work when having `jfxMainAppJarName = "build/libs/purchaseList.jar"` not being set?

**Comment by lgLindstrom** [comment]
Thanks,,    now jfxRun it works.

Unfortunately I have another problem... this time with jfxNative. Added output from terminal window.
( seems like terminal gives  characters that are interpreted as "strike through" )
For me it seems like Ino are missing some files. I've used file explore and checked directories where they supposed to be,,,EMPTY.


```
--------
10:49:47: Executing task 'jfxNative'...

:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:jar UP-TO-DATE
:jfxJar
:jfxNative
The jar lib\weld-se-core-2.4.6.Final.jar has a main class org.jboss.weld.environment.se.StartMain that does not match the declared main com.consius.purchaseList.MainApplication
The jar lib\icu4j-57.1.jar has a main class com.ibm.icu.util.VersionInfo that does not match the declared main com.consius.purchaseList.MainApplication
Running [C:\Program Files\Java\jdk-9.0.4\bin\java.exe, -version]
Creating app bundle: Purchase List Generator in C:\HostedProjects\PurchaseListGenerator\build\jfx\native
"Adding modules: [java.rmi, java.sql, javafx.web, jdk.charsets, java.logging, java.xml.crypto, java.xml, jdk.xml.dom, jdk.jfr, java.datatransfer, jdk.packager.services, jdk.httpserver, javafx.base, jdk.net, java.desktop, java.naming, javafx.controls, java.prefs, java.security.sasl, jdk.naming.rmi, jdk.zipfs, java.base, jdk.crypto.ec, jdk.management.agent, java.management, java.sql.rowset, javafx.swing, jdk.jsobject, jdk.sctp, java.smartcardio, jdk.unsupported, jdk.jdwp.agent, jdk.scripting.nashorn, java.instrument, java.security.jgss, jdk.management, java.compiler, javafx.graphics, jdk.security.auth, java.scripting, javafx.fxml, jdk.dynalink, javafx.media, jdk.accessibility, java.management.rmi, jdk.naming.dns, jdk.security.jgss, jdk.localedata] to runtime image."
outputDir = C:\HostedProjects\PurchaseListGenerator\build\jfx\native\Purchase List Generator\runtime
modulePath = [C:\Program Files\Java\jdk-9.0.4\jmods]
addModules = [java.rmi, java.sql, javafx.web, jdk.charsets, java.logging, java.xml.crypto, java.xml, jdk.xml.dom, jdk.jfr, java.datatransfer, jdk.packager.services, jdk.httpserver, javafx.base, jdk.net, java.desktop, java.naming, javafx.controls, java.prefs, java.security.sasl, jdk.naming.rmi, jdk.zipfs, java.base, jdk.crypto.ec, jdk.management.agent, java.management, java.sql.rowset, javafx.swing, jdk.jsobject, jdk.sctp, java.smartcardio, jdk.unsupported, jdk.jdwp.agent, jdk.scripting.nashorn, java.instrument, java.security.jgss, jdk.management, java.compiler, javafx.graphics, jdk.security.auth, java.scripting, javafx.fxml, jdk.dynalink, javafx.media, jdk.accessibility, java.management.rmi, jdk.naming.dns, jdk.security.jgss, jdk.localedata]
limitModules = []
excludeFileList = .*\.diz
stripNativeCommands = true
userArguments = {}
Using default package resource [icon]  (add package/windows/Purchase List Generator.ico to the class path to customize)
Using default package resource [Template for creating executable properties file.]  (add package/windows\PurchaseListGenerator.properties to the class path to customize)
Running [reg, query, "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows Defender\Real-Time Protection", /v, DisableRealtimeMonitoring]
Running [reg, query, "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows Defender\Exclusions\Paths"]
Warning: Windows Defender may prevent the Java Packager from functioning. If there is an issue, it can be addressed by either disabling realtime monitoring, or adding an exclusion for the directory "C:\Users\LARS-G~2\AppData\Local\Temp\".
Running [C:\Program Files\Java\jdk-9.0.4\bin\javapackager.exe, --icon-swap, C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler5495132943177703272\windows\PurchaseListGenerator.ico, C:\HostedProjects\PurchaseListGenerator\build\jfx\native\Purchase List Generator\PurchaseListGenerator.exe]
Icon File Name: C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler5495132943177703272\windows\PurchaseListGenerator.ico
Executable File Name: C:\HostedProjects\PurchaseListGenerator\build\jfx\native\Purchase List Generator\PurchaseListGenerator.exe
Running [C:\Program Files\Java\jdk-9.0.4\bin\javapackager.exe, --version-swap, C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler5495132943177703272\windows\PurchaseListGenerator.properties, C:\HostedProjects\PurchaseListGenerator\build\jfx\native\Purchase List Generator\PurchaseListGenerator.exe]
Resource File Name: C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler5495132943177703272\windows\PurchaseListGenerator.properties
Executable File Name: C:\HostedProjects\PurchaseListGenerator\build\jfx\native\Purchase List Generator\PurchaseListGenerator.exe
Exception: java.nio.file.NoSuchFileException: C:\HostedProjects\PurchaseListGenerator\build\jfx\app\lib\javax.inject-1.jar -> C:\HostedProjects\PurchaseListGenerator\build\jfx\native\Purchase List Generator\app\lib\javax.inject-1.jar
Config files are saved to C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler5495132943177703272\windows. Use them to customize package.
Result application bundle: C:\HostedProjects\PurchaseListGenerator\build\jfx\native
The jar lib\weld-se-core-2.4.6.Final.jar has a main class org.jboss.weld.environment.se.StartMain that does not match the declared main com.consius.purchaseList.MainApplication
The jar lib\icu4j-57.1.jar has a main class com.ibm.icu.util.VersionInfo that does not match the declared main com.consius.purchaseList.MainApplication
Running [C:\Program Files\Java\jdk-9.0.4\bin\java.exe, -version]
Running [C:\Program Files (x86)\Inno Setup 5\iscc.exe, /?]
  Detected [C:\Program Files (x86)\Inno Setup 5\iscc.exe] version [5]
Running [reg, query, "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows Defender\Real-Time Protection", /v, DisableRealtimeMonitoring]
Running [reg, query, "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows Defender\Exclusions\Paths"]
Warning: Windows Defender may prevent the Java Packager from functioning. If there is an issue, it can be addressed by either disabling realtime monitoring, or adding an exclusion for the directory "C:\Users\LARS-G~2\AppData\Local\Temp\".
"Adding modules: [java.rmi, java.sql, javafx.web, jdk.charsets, java.logging, java.xml.crypto, java.xml, jdk.xml.dom, jdk.jfr, java.datatransfer, jdk.packager.services, jdk.httpserver, javafx.base, jdk.net, java.desktop, java.naming, javafx.controls, java.prefs, java.security.sasl, jdk.naming.rmi, jdk.zipfs, java.base, jdk.crypto.ec, jdk.management.agent, java.management, java.sql.rowset, javafx.swing, jdk.jsobject, jdk.sctp, java.smartcardio, jdk.unsupported, jdk.jdwp.agent, jdk.scripting.nashorn, java.instrument, java.security.jgss, jdk.management, java.compiler, javafx.graphics, jdk.security.auth, java.scripting, javafx.fxml, jdk.dynalink, javafx.media, jdk.accessibility, java.management.rmi, jdk.naming.dns, jdk.security.jgss, jdk.localedata] to runtime image."
outputDir = C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator\runtime
modulePath = [C:\Program Files\Java\jdk-9.0.4\jmods]
addModules = [java.rmi, java.sql, javafx.web, jdk.charsets, java.logging, java.xml.crypto, java.xml, jdk.xml.dom, jdk.jfr, java.datatransfer, jdk.packager.services, jdk.httpserver, javafx.base, jdk.net, java.desktop, java.naming, javafx.controls, java.prefs, java.security.sasl, jdk.naming.rmi, jdk.zipfs, java.base, jdk.crypto.ec, jdk.management.agent, java.management, java.sql.rowset, javafx.swing, jdk.jsobject, jdk.sctp, java.smartcardio, jdk.unsupported, jdk.jdwp.agent, jdk.scripting.nashorn, java.instrument, java.security.jgss, jdk.management, java.compiler, javafx.graphics, jdk.security.auth, java.scripting, javafx.fxml, jdk.dynalink, javafx.media, jdk.accessibility, java.management.rmi, jdk.naming.dns, jdk.security.jgss, jdk.localedata]
limitModules = []
excludeFileList = .*\.diz
stripNativeCommands = true
userArguments = {}
Using default package resource [icon]  (add package/windows/Purchase List Generator.ico to the class path to customize)
Using default package resource [Template for creating executable properties file.]  (add package/windows\PurchaseListGenerator.properties to the class path to customize)
Running [reg, query, "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows Defender\Real-Time Protection", /v, DisableRealtimeMonitoring]
Running [reg, query, "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows Defender\Exclusions\Paths"]
Warning: Windows Defender may prevent the Java Packager from functioning. If there is an issue, it can be addressed by either disabling realtime monitoring, or adding an exclusion for the directory "C:\Users\LARS-G~2\AppData\Local\Temp\".
Running [C:\Program Files\Java\jdk-9.0.4\bin\javapackager.exe, --icon-swap, C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\windows\PurchaseListGenerator.ico, C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator\PurchaseListGenerator.exe]
Icon File Name: C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\windows\PurchaseListGenerator.ico
Executable File Name: C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator\PurchaseListGenerator.exe
Running [C:\Program Files\Java\jdk-9.0.4\bin\javapackager.exe, --version-swap, C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\windows\PurchaseListGenerator.properties, C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator\PurchaseListGenerator.exe]
Resource File Name: C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\windows\PurchaseListGenerator.properties
Executable File Name: C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator\PurchaseListGenerator.exe
Exception: java.nio.file.NoSuchFileException: C:\HostedProjects\PurchaseListGenerator\build\jfx\app\lib\javax.inject-1.jar -> C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator\app\lib\javax.inject-1.jar
Config files are saved to C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\windows. Use them to customize package.
Using default package resource [Inno Setup project file]  (add package/windows/Purchase List Generator.iss to the class path to customize)
Using default package resource [setup dialog icon]  (add package/windows/Purchase List Generator-setup-icon.bmp to the class path to customize)
Using default package resource [script to run after application image is populated]  (add package/windows/Purchase List Generator-post-image.wsf to the class path to customize)
Generating EXE for installer to: C:\HostedProjects\PurchaseListGenerator\build\jfx\native
Running [C:\Program Files (x86)\Inno Setup 5\iscc.exe, /oC:\HostedProjects\PurchaseListGenerator\build\jfx\native, C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator.iss] in C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image
Inno Setup 5 Command-Line Compiler
Copyright (C) 1997-2016 Jordan Russell. All rights reserved.
Portions Copyright (C) 2000-2016 Martijn Laan
Inno Setup Preprocessor
Copyright (C) 2001-2004 Alex Yackimoff. All rights reserved.

Compiler engine version: Inno Setup 5.5.9 (a)

[ISPP] Preprocessing.
[ISPP] Preprocessed.

Parsing [Setup] section, line 4
Parsing [Setup] section, line 5
Parsing [Setup] section, line 6
Parsing [Setup] section, line 7
Parsing [Setup] section, line 8
Parsing [Setup] section, line 9
Parsing [Setup] section, line 10
Parsing [Setup] section, line 14
Parsing [Setup] section, line 15
Parsing [Setup] section, line 16
Parsing [Setup] section, line 17
Parsing [Setup] section, line 18
Parsing [Setup] section, line 19
Parsing [Setup] section, line 20
Parsing [Setup] section, line 21
Parsing [Setup] section, line 23
Parsing [Setup] section, line 25
Parsing [Setup] section, line 26
Parsing [Setup] section, line 27
Parsing [Setup] section, line 28
Parsing [Setup] section, line 29
Parsing [Setup] section, line 30
Parsing [Setup] section, line 31
Parsing [Setup] section, line 32
Parsing [Setup] section, line 33
Parsing [Setup] section, line 34
Parsing [Setup] section, line 35
Reading file (WizardImageFile)
   File: C:\Program Files (x86)\Inno Setup 5\WIZMODERNIMAGE.BMP
Reading file (WizardSmallImageFile)
   File: C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator-setup-icon.bmp
Preparing Setup program executable
   Updating icons (SETUP.E32)
Reading default messages from Default.isl
Parsing [Languages] section, line 39
   File: C:\Program Files (x86)\Inno Setup 5\Default.isl
Parsing [LangOptions], [Messages], and [CustomMessages] sections
   Messages in script file
Reading [Code] section
Parsing [Icons] section, line 46
Parsing [Icons] section, line 47
Parsing [Run] section, line 51
Parsing [Run] section, line 52
Parsing [Run] section, line 53
Parsing [UninstallRun] section, line 56
Parsing [Files] section, line 42
Error on line 42 in C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator.iss: Source file "C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator\Purchase List Generator.exe" does not exist.
Compile aborted.
java.io.IOException: Exec failed with code 2 command [[C:\Program Files (x86)\Inno Setup 5\iscc.exe, /oC:\HostedProjects\PurchaseListGenerator\build\jfx\native, C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image\Purchase List Generator.iss] in C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\images\win-exe.image
	at jdk.packager/com.oracle.tools.packager.IOUtils.exec(IOUtils.java:169)
	at jdk.packager/com.oracle.tools.packager.IOUtils.exec(IOUtils.java:142)
	at jdk.packager/com.oracle.tools.packager.IOUtils.exec(IOUtils.java:136)
	at jdk.packager/com.oracle.tools.packager.windows.WinExeBundler.buildEXE(WinExeBundler.java:703)
	at jdk.packager/com.oracle.tools.packager.windows.WinExeBundler.bundle(WinExeBundler.java:372)
	at jdk.packager/com.oracle.tools.packager.windows.WinExeBundler.execute(WinExeBundler.java:174)
	at de.dynamicfiles.projects.gradle.plugins.javafx.tasks.workers.JfxNativeWorker.jfxnative(JfxNativeWorker.java:348)
	at de.dynamicfiles.projects.gradle.plugins.javafx.tasks.JfxNativeTask.jfxnative(JfxNativeTask.java:32)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:46)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:780)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:747)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:121)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:110)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:92)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:70)
	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:62)
	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:60)
	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:97)
	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:87)
	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:52)
	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
	at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
	at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker$1.run(DefaultTaskGraphExecuter.java:248)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:241)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:230)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.processTask(DefaultTaskPlanExecutor.java:123)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.access$200(DefaultTaskPlanExecutor.java:79)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:104)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:98)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.execute(DefaultTaskExecutionPlan.java:626)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.executeWithTask(DefaultTaskExecutionPlan.java:581)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.run(DefaultTaskPlanExecutor.java:98)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.base/java.lang.Thread.run(Thread.java:844)
  Config files are saved to C:\Users\LARS-G~2\AppData\Local\Temp\fxbundler8118181887775257889\windows. Use them to customize package.
The jar lib\weld-se-core-2.4.6.Final.jar has a main class org.jboss.weld.environment.se.StartMain that does not match the declared main com.consius.purchaseList.MainApplication
The jar lib\icu4j-57.1.jar has a main class com.ibm.icu.util.VersionInfo that does not match the declared main com.consius.purchaseList.MainApplication
Running [C:\Program Files\Java\jdk-9.0.4\bin\java.exe, -version]
Detected [candle.exe] version 0 but version 3 is required.
Detected [light.exe] version 0 but version 3 is required.

BUILD SUCCESSFUL in 19s
5 actionable tasks: 2 executed, 3 up-to-date
10:50:08: Task execution finished 'jfxNative'.


```

**Comment by FibreFoX** [comment]
What really is confusing:
```
Detected [candle.exe] version 0 but version 3 is required.
Detected [light.exe] version 0 but version 3 is required.
```
Looks like some non compatible version here being installed.

**Comment by lgLindstrom** [comment]
Any advise to clean up my installation?

**Comment by FibreFoX** [comment]
Hehe, I got confused by your output, sorry. These `but version 3 is required`-messages are related to creating MSI-installers, False alert ;)

Maybe it's something with having whitespaces inside the launcher name, just a wild guess. Try to create a separate project, similar to your real project and try to re-create it, often it's a combination of things, or just mis-configuration.

Nevertheless I will close this issue, we are getting a bit offtopic, but don't hesitate to report your findings here. Closing issues helps me to get the overview back.
